integration/presto/src/test/scala/org/apache/carbondata/presto/integrationtest/PrestoTestNonTransactionalTableFiles.scala

```diff
@@ -176,24 +188,35 @@ class PrestoTestNonTransactionalTableFiles extends FunSuiteLike with BeforeAndAf
   def buildTestDataOtherDataType(rows: Int, sortColumns: Array[String]): Any = {
     val fields: Array[Field] = new Array[Field](5)
     // same column name, but name as boolean type
-    fields(0) = new Field("name", DataTypes.BOOLEAN)
+    fields(0) = new Field("name", DataTypes.VARCHAR)
     fields(1) = new Field("age", DataTypes.INT)
-    fields(2) = new Field("id", DataTypes.BYTE)
+    fields(2) = new Field("id", DataTypes.BINARY)
     fields(3) = new Field("height", DataTypes.DOUBLE)
     fields(4) = new Field("salary", DataTypes.FLOAT)
 
+    val imagePath = "../../sdk/sdk/src/test/resources/image/carbondatalogo.jpg"
```

Comment:
better to base on rootPath
val imagePath = "$rootPath/sdk/sdk/src/test/resources/image/carbondatalogo.jpg"

another question: why presto module need scala?

Reply:
> another question: why presto module need scala?

Reply:
ok, used root path